embassy-nrf/src/uarte.rs

```diff
@@ -929,6 +762,75 @@ impl<'d, U: Instance, T: TimerInstance> UarteRxWithIdle<'d, U, T> {
         Ok(n)
     }
 }
+
```

Comment:
Nothing changed in the following two functions - just moved to help the code layout a bit.